assignment3/graph.dart

```diff
@@ -0,0 +1,171 @@
+import 'dart:io';
+import 'util.dart';
+import 'functions.dart';
+
+class Graph {
+  int nodeId = 0;
+  String nodeName = "";
+  Set<int> immediateChildren = {};
+  Set<int> immediateParent = {};
+
+  Map<dynamic, dynamic> additionalDetails = {};
+  static Set<int> uniqueId = {};
+  static Map<int, Graph> allExistingNodes = {};
+
+  Graph({required this.nodeId, required this.nodeName});
```

Comment:
Where's additional information about the Node?

Reply:
additional info is optional . user can enter it manually should i make it mendatory?